### PR TITLE
config parameters not used for anything

### PR DESCRIPTION
**Pull request by star-gazer111**

### CODE REVIEW

**Comment by star-gazer111** [comment]
fix #818

File: roles/pool/config-examples/pool-config-local-tp-example.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # SRI Pool config
 authority_public_key = "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72"
 authority_secret_key = "mkDLTBBRxdBv998612qipDYoTK3YUrqLe8uWw7gu3iXbSrn2n"
-#authority_secret_key = "7qbpUjScc865jyX2kiB4NVJANoC7GA7TAJupdzXWkc62"
+authority_secret_key = "7qbpUjScc865jyX2kiB4NVJANoC7GA7TAJupdzXWkc62"
```
Comment (plebhash) [inline roles/pool/config-examples/pool-config-local-tp-example.toml:4]:
```suggestion
```

**Comment by plebhash** [comment]
@star-gazer111 we're merging PRs against `dev` branch now

I tried editing your PR to make it against `dev` but it's introducing many unnecessary commits after this

please start your development from `dev` branch and open a new PR against `dev`

**Comment by star-gazer111** [comment]
> @star-gazer111 we're merging PRs against `dev` branch now
> 
> I tried editing your PR to make it against `dev` but it's introducing many unnecessary commits after this
> 
> please start your development from `dev` branch and open a new PR against `dev`

okay, I reopened it here #838

**Pull request closed without merging**